package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fixtable-ember",
-  "version": "0.2.2",
+  "version": "0.3.2",
```

Comment:
How do you anticipate that this will break the existing contract? I'm not sure it will, but if it does, we should go to version 1.0.0 (i.e., do a full version bump).